website/src/content/development/page-content-layouts.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: Page Content Layouts
+navLevels: 2
+---
+
+At commercetools, one core design principle is a visual hierarchy, having a clean and usable user interface using spacings. As part of these guidelines, it's essential to properly structure a page's overall content.
+
+Layouts are the foundation for your page content structure. They provide the necessary flexibility to build beautiful and accessible user interfaces while adding constraints to help you create a familiar user experience.
+
+For example, implementing a details page might require rendering form elements and additional meta information, while a list page with a large table requires a different layout. Layouts ensure your page content has the correct hierarchy and structure for the best user experience for the given use case.
+
+
+<Info>
+
+This feature is available from version `21.22.0` onwards.
+
+</Info>
+
+# Layout types
+
+A layout sets the container's minimum and maximum width and position on the page.
+To support various page content requirements, we provide the following layout types:
+
+| Layout type | Use case |
+|-------------|----------|
+| [Narrow](#narrow) | Simple forms, collapsible panels, and cards. |
+| [Wide (single column)](#narrow) | Large forms or tables with few columns. |
+| [Wide (Two columns 1:1)](#narrow) | Content grouped together to save vertical space. |
+| [Wide (Two columns 2:1)](#narrow) | Wider tables and some meta information. |
+| [Full](#full) | Horizontal reading flow such as large tables. |
+
+## Narrow
+
+A narrow layout consists of a single-column with a constrained width size.
+
+![Layout narrow](/images/page-content-layout-narrow.png)
+
+To implement a narrow layout, use the [`<PageContentNarrow>`](/api-reference/commercetools-frontend-application-components#pagecontentnarrow) component.
+
+```jsx
+import { PageContentNarrow } from '@commercetools-frontend/application-components';
+
+const ChannelsDetails = () => {
+  // ...
+  return (
+    <PageContentNarrow>
+      <Spacings.Stack>
+        <div>Title</div>
+        <ChannelsForm />
+      </Spacings.Stack>
+    </PageContentNarrow>
+  )
+};
+```
+
+### Best practices
+
+* Use a narrow layout for simple forms where form fields must span across the available width (`horizontalConstraint: scale`).
```

Comment:
I remember someone from the writers once told me to avoid starting a section with a bullet list.
That's why I added the short sentence `This layout should:`.

To give an example, see Polaris docs

<img width="244" alt="image" src="https://user-images.githubusercontent.com/1110551/236758644-eee92041-3afa-4a58-a6ed-9faf940ca1f0.png">

```suggestion
### Best practices

This layout should:
* Use a narrow layout for simple forms where form fields must span across the available width (`horizontalConstraint: scale`).
```

Reply:
Got it, good to know 🤗